dev-cpp/ms-gsl/ms-gsl-0_pre20180304-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
+
+DESCRIPTION="Guideline Support Library implementation by Microsoft"
+HOMEPAGE="https://github.com/Microsoft/GSL"
+SRC_URI="https://github.com/Microsoft/gsl/tarball/6a33b97a84f9c0a60ede78b5db98647e9a48d6c9 -> ${P}.tar.gz"
```

Comment:
I presume the only change here is adding the patch, so I won't request changing this but for the next bump, please switch to GitHub's `archive` URL.